src/frames/UtilFrame.java

```diff
@@ -564,6 +570,20 @@ private void jComboBox1ActionPerformed() {//GEN-FIRST:event_jButton4ActionPerfor
                 ArrayVisualizer.setComparator(2);
                 break;
 
+            case "Sorting Networks":
+                if (ArrayVisualizer.enableBenchmarking(false))
+                    break;
+                    jButton6.setEnabled(true);
+                    ArrayVisualizer.setComparator(4);
+                if (!SortingNetworkGenerator.verifyPythonVersionAndDialog())
+                    jComboBox1.setSelectedIndex(0); // Failure to find Python installation
+                if (ArrayVisualizer.getCurrentLength() >= 512) {
+                    JOptionPane.showMessageDialog(null, "Array lengths greater than 512 can take a really long time to generate.",
+                        "Sorting Network Visualizer", JOptionPane.WARNING_MESSAGE);
+                    ArrayVisualizer.getArrayFrame().setLengthSlider(256);
```

Comment:
Feels kinda awkward to bump the number to something way past the limit (ie, if I set the length to 257-511, I would not get the error and the length would remain above 256).

Reply:
@aphitorite solved this now. (I think. I might misunderstand what you mean.)